src/Module/Item/Compose.php

```diff
@@ -241,6 +240,9 @@ protected function content(array $request = []): string
 				'deny_cid'  => $contact_deny_list,
 				'deny_gid'  => $group_deny_list,
 			]),
+
+			// Dropzone
+			'$max_imagesize' => round(\Friendica\Util\Strings::getBytesFromShorthand(DI::config()->get('system', 'maximagesize')) / 1000000,1),
```

Comment:
Code standards: Please add a space after commas.

Additionally, you can use the local `config` variable.
```suggestion
			'$max_imagesize' => round(\Friendica\Util\Strings::getBytesFromShorthand($this->config->get('system', 'maximagesize')) / 1000000, 1),
```